src/templates/staff/roster.html

```diff
@@ -0,0 +1,253 @@
+   * the table.
+   */
+  function updateStaffTable() {
+    $.get("{{ url_for('.get_course_staff_roster', cid=course._id) }}", (data) => {
+      $("#staff-table").empty()
+      $(staff_alert).hide()
+      // Add admins to table first
+      let admin_ids = data.admin_ids;
+      for (let i = 0; i < admin_ids.length; i++) {
+        let firstItemOnlyClass = i === 0 ? 'border-top-0' : '';
+        let demoteDisableOrNotClass = admin_ids[i] === data.user ? 'text-muted' : 'clickable text-info demote-admin';
+        let deleteDisableOrNotClass = admin_ids[i] === data.user ? 'text-muted' : 'clickable text-danger remove-staff';
+        $("#staff-table").append(`
+          <li class="list-group-item ${firstItemOnlyClass}">
+            <h6 class="font-weight-bold font-italic d-inline">${admin_ids[i]}</h6>
+            <div class="float-right">
+              <i class="${demoteDisableOrNotClass} fas fa-lg fa-arrow-circle-down mr-2"
+                 data-netid="${admin_ids[i]}"></i>
+              <i class="${deleteDisableOrNotClass} fas fa-lg fa-trash-alt" data-netid="${admin_ids[i]}"></i>
+            </div>
+          </li>`
+        )
+      }
+      // Add staffs to table second
+      let staff_ids = data.staff_ids;
+      for (let i = 0; i < staff_ids.length; i++) {
+        if (admin_ids.includes(staff_ids[i])) {
+          continue;
+        }
+        $("#staff-table").append(`
+          <li class="list-group-item">${staff_ids[i]}
+            <div class="float-right">
+              <i class="clickable fas fa-lg promote-staff fa-arrow-circle-up text-success mr-2"
+                 data-netid="${staff_ids[i]}"></i>
+              <i class="clickable remove-staff fas fa-lg fa-trash-alt text-danger" data-netid="${staff_ids[i]}"></i>
+            </div>
+          </li>
+        `)
+      }
+      // Add event listeners
+      $(".remove-staff").click(removeStaff)
+      $(".promote-staff").click(promoteStaff)
+      $(".demote-admin").click(demoteAdmin)
+    }).fail((jqXHR) => {
+      console.log(jqXHR)
+      showError("Something went wrong, please refresh the page.", staff_alert)
+    })
+  }
+
+  /**
+   * Update student table in a way similar to updateStaffTable.
+   */
+  function updateStudentTable() {
+    $.get("{{ url_for('.get_course_student_roster', cid=course._id) }}", (students) => {
+      $("#student-table").empty()
+      $(student_alert).hide()
+      for (let i = 0; i < students.length; i++) {
+        let firstItemOnlyClass = i === 0 ? 'border-top-0' : '';
+        $("#student-table").append(`
+          <li class="list-group-item ${firstItemOnlyClass}">${students[i]}
+            <div class="float-right">
+              <i class="clickable remove-student fas fa-lg fa-trash-alt text-danger" data-netid="${students[i]}"></i>
+            </div>
+          </li>
+        `)
+      }
+      $(".remove-student").click(removeStudent)
+      // update the "global" array for search
+      student_ids = students
```

Comment:
I know we don't have linting, but could we add semicolons to all the lines that don't have them, for the sake of standards?